src/lib/common/string.cpp

```diff
@@ -1026,6 +1026,15 @@ template <> std::string toString(double f)
 {
   std::ostringstream ss;
 
+  // We should test for each case to return the correct string.
+  if (f != f) {
```

Comment:
since we're now including math.h, why not use `isnan()`? (`f != f` is correct, I know, but seems less explicit than `isnan()`)

Reply:
Sure! (Same for the other comment)